README.md

```diff
@@ -28,10 +28,10 @@ npm run apply-coding-standards
 COMPOSE_PROJECT_NAME=aapodwalk
 COMPOSE_DOMAIN=aapodwalk.local.itkdev.dk
 
-REACT_APP_API_BASE=API_URL_HERE # most probably https://aapodwalk-api.local.itkdev.dk/
-REACT_APP_API_ROUTE=api/v1/
-REACT_APP_TOKEN=token_stuff_here # created in api (123?)
+VITE_APP_API_BASE=API_URL_HERE # most probably https://aapodwalk-api.local.itkdev.dk/
+VITE_APP_API_ROUTE=api/v1/
+VITE_APP_TOKEN=token_stuff_here # created in api (123?)
 
-REACT_APP_DF_MAP_USERNAME=username_here # can be found in 1password
-REACT_APP_DF_MAP_PASSWORD=password_here # can likewise be found in 1password
+VITE_APP_DF_MAP_USERNAME=username_here # can be found in 1password
+VITE_APP_DF_MAP_PASSWORD=password_here # can likewise be found in 1password
```

Comment:
Do we really need the name of something we use to build the project in variable names? Couldn't we just use `APP_` as the prefix?

Reply:
<img width="713" alt="image" src="https://github.com/itk-dev/aapodwalk/assets/15377965/00a6d698-00d5-4d4f-9e44-5168995528f8">

If I understand this correct, then yes. 

https://vitejs.dev/guide/env-and-mode.html#env-files